conf/__init__.py

```diff
@@ -45,6 +45,9 @@
 
 # IDEX Tests
 idex_api_key = os.getenv("IDEX_API_KEY")
+test_idex_erc20_token_address_1 = os.getenv("IDEX_TOKEN_ADDRESS_1")
+test_idex_erc20_token_address_2 = os.getenv("IDEX_TOKEN_ADDRESS_2")
```

Comment:
@chowryan I don't think these token addresses are actually used in idex tests.
Only the ticker?

Reply:
dont understand your question. why are you adding this?

Reply:
@chowryan added those because the original tests refer to:

```
erc20_token_addresses=[conf.test_ddex_erc20_token_address_1,
conf.test_ddex_erc20_token_address_2],
```


https://github.com/CoinAlpha/hummingbot/blob/4941ec38b8090b8f458d19ca5ad98d667f561b4e/test/integration/test_idex_market.py#L84

Maybe this is a carry over from copying over the ddex tests, that wasn't updated or cleaned up.

I added these to conf so the variables (that are referenced in the test) need to be sourced from the environment variables to run in jenkins.

Therefore my question, are these actually being used in the tests? (they seem to be not)
![image](https://user-images.githubusercontent.com/12274903/65646676-69d20600-e02e-11e9-8dbf-5171ad5650df.png)